<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWidget</class>
 <widget class="QWidget" name="DashboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>945</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>14</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topBar">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="menuButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/images/resources/images/dashboard/menu.png</normaloff>:/images/resources/images/dashboard/menu.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>13</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dashboard">
       <property name="text">
        <string>Dashboard</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="logo">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="main.qrc">:/images/resources/images/dashboard/logo.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="minimize">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="main.qrc">:/images/resources/images/dashboard/minimize.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="exit">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="main.qrc">:/images/resources/images/dashboard/exit.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="yourApps" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16px 'Roboto';
</string>
      </property>
      <property name="text">
       <string>Your Apps</string>
      </property>
     </widget>
     <widget class="QWidget" name="gradientLine_1" native="true">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>27</y>
        <width>50</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QWidget {
	background-color: qlineargradient(x1:0 y1:0, x2:1 y2:0, stop:1 rgba(44, 95, 255, 1), stop:0 rgba(196, 148, 234, 1));
}</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Server App Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DashboardApp" name="app1" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>74</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>74</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DashboardApp" name="app2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>74</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>74</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DashboardApp" name="app3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="yourMachines" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16px 'Roboto';
</string>
      </property>
      <property name="text">
       <string>Your Machines</string>
      </property>
     </widget>
     <widget class="QWidget" name="gradientLine_2" native="true">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>27</y>
        <width>50</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QWidget {
	background-color: qlineargradient(x1:0 y1:0, x2:1 y2:0, stop:1 rgba(44, 95, 255, 1), stop:0 rgba(196, 148, 234, 1));
}</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="DashboardMachine" name="machine1" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DashboardMachine" name="machine2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DashboardMachine" name="machine3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vSpacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DashboardApp</class>
   <extends>QWidget</extends>
   <header>dashboardapp.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DashboardMachine</class>
   <extends>QWidget</extends>
   <header>dashboardmachine.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
